src/test/java/edu/harvard/iq/dataverse/api/HarvestingClientsIT.java

```diff
@@ -54,14 +58,25 @@ public static void setUpClass() {
         
     }
     @AfterEach
-    public void cleanup() {
+    public void cleanup() throws InterruptedException {
         if (clientApiPath != null) {
             Response deleteResponse = given()
                     .header(UtilIT.API_TOKEN_HTTP_HEADER, adminUserAPIKey)
                     .delete(clientApiPath);
             clientApiPath = null;
             System.out.println("deleteResponse.getStatusCode(): " + deleteResponse.getStatusCode());
+
+            int i = 0;
+            int maxWait = 20;
+            String query = "dsPersistentId:" + globalIdList.stream().map(s -> "\""+s+"\"").collect(Collectors.joining(","));
```

Comment:
Why was this necessary, btw - what was the problem with ```"metadataSource:" + nickName```? 

Doesn't matter really, as long as it works. But I can't help but feel like there's too much going on in this test class, vs. its useful value. (Strictly speaking, there is no need to harvest the first 7 datasets twice in a row; that second set could just contain the 1 dataset with the CVV, maybe?)

Reply:
I'm still *mildly* curious why it was necessary to search by individual dataset ids here.